docs/reference/search/profile.asciidoc

```diff
@@ -53,72 +53,90 @@ This will yield the following result:
       "failed": 0
    },
    "hits": {
-      "total": 1,
-      "max_score": 1,
+      "total": 3,
+      "max_score": 1.078072,
```

Comment:
I copy/pasted a response to get all the new elements (didn't want to make up the counts), but then the timings no longer matched and my OCD wouldn't allow mismatching timings all over the page :)